interceptors/ratelimit/ratelimit.go

```diff
@@ -22,7 +22,7 @@ type Limiter interface {
 func UnaryServerInterceptor(limiter Limiter) grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req any, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (any, error) {
 		if err := limiter.Limit(ctx); err != nil {
-			return nil, status.Errorf(codes.ResourceExhausted, "%s is rejected by grpc_ratelimit middleware, please retry later. %s", info.FullMethod, err)
+			return nil, status.Errorf(codes.ResourceExhausted, "%s is rejected by grpc_ratelimit middleware, please retry later. %v", info.FullMethod, err)
```

Comment:
I don't think this is an improvement - the value in using specific verbs (like `%s`) over generic (like `%v`) is that go vet can warn when you're using the specific verbs incorrectly. Please revert this change.

Reply:
Hm, I'm not sure I understand your concern. Isn't it idiomatic to use `%v` for the `error` type formatting? 99% of code does it that way, including in this codebase.

Reply:
At the same time, there is code that does the opposite of what you are suggesting - uses `%v` for e.g. `float32`:

https://github.com/ash2k/go-grpc-middleware/blob/0df73fc576befff7acf6f397634a64c54aa4ad11/interceptors/logging/options.go#L81-L94

Reply:
I don't think that's an argument against what I'm saying, it'd be great to change those instances to not use %v either. I would only use %v where I truly don't know the type.

Reply:
I'm not sure if you've seen my previous comment? `%v` is always used for errors (I mean, check stdlib?), it even handles them in a [special way](https://pkg.go.dev/fmt):

```
4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any). 
```

Reply:
That's not a special case for the %v verb, it goes for any verb. All the more reason to use %s for errors, so we get the benefit of `go vet` checking our interpolations for us.

Reply:
That's interesting! I didn't know this special case is not just for `error`. Ok, I changed it back to `%s`.